src/mono/wasm/debugger/DebuggerTestSuite/MiscTests.cs

```diff
@@ -872,7 +872,7 @@ public async Task GetSourceEmbeddedSource()
                 await CheckProps(t_props, new
                     {
                         Status = TGetter("Status")
-                    }, "t_props", num_fields: 54);
+                    }, "t_props", num_fields: 58);
```

Comment:
You can replace `num_fields` by `skip_num_fields_check: true`.

Reply:
On the one hand, yes but on the other: when we break something so that we won't be displaying all the fields correctly (e.g. I would roll back this PR) then this test will help us because it will fail. I would prefer to leave it, if someone will find it problematic at some point - it can be replaced then.